heudiconv/dicoms.py

```diff
@@ -20,6 +20,7 @@
 
 lgr = logging.getLogger(__name__)
 total_files = 0
+compresslevel = 9
```

Comment:
ha -- a cute workaround!  I don't mind accepting.

I only worry that eventually it can get moved thus breaking this "feature" for you unless we 
1. at least annotate it here
```suggestion
# Might be monkey patched by user heuristic to tune desired compression level
# Preferably do not move/rename unless 
compresslevel = 9
```
2. in your heuristic you better first check `if hasattr(dicoms, compresslevel)` before assigning or someone moves it away -- you might be out of luck.

sure thing this is just a quite an ad-hoc workaround. another one could have been for you to just overload `tarfile.open` and provide your custom compresslevel there... smth like

```
import tarfile

def _tarfile_open(*args, **kwargs):
    kwargs.set_default('compresslevel', 3)
    return tarfile.open(*args, **kwargs)

tarfile.open = _tarfile_open
```
or alike.  Just an idea